app/views/catalog/_cocina_default.html.erb

```diff
@@ -6,10 +6,15 @@
   </h2>
   <div id="document-cocina-section" class="accordion-collapse collapse" aria-labelledby="document-cocina-heading">
     <div class="accordion-body" data-controller="json-renderer">
+      <div class="argo-show-btn-group">
```

Comment:
Is this class necessary?

Reply:
@jcoyne Necessary? No. It made the buttons layout a little nicer to my eyes, though.

Reply:
If we don't have any styles or actions affiliated with this class, then I think it causes unnecessary cognitive load and it should be removed.

Reply:
@jcoyne Should I remove the div entirely then or use an empty div?

Reply:
I think an empty div is fine if you want to group those.

Reply:
@jcoyne OK, done